### DISCUSSION TITLE
Failed to write database table in the next generation of the Wavelog Installer from HB9HIL

### DISCUSSION THREAD

**Discussion by BG6HJE-edu** [wavelog/wavelog#618] [Q&A]
Hi, this is BG6HJE. I am getting “Failed to write database table” during installation. 
I use the code from [this version from HB9HIL](https://github.com/HB9HIL/wavelog/tree/installer_v2)
I would like to know how to solve this problem.
![error screenshot](https://github.com/user-attachments/assets/24c05497-14bf-41cc-a1ce-584c72c1eed0)

**Comment by HB9HIL** [comment]
Hello @BG6HJE-edu 

Did you drop and recreate the database before you started the installer?

> [!CAUTION]
> This Installer is currently in testing and is not part of the official Wavelog Repository yet. See PR #603

**Reply by BG6HJE-edu** [reply]
Thanks for your reply. I just create a new database before installing.

**Reply by HB9HIL** [reply]
1. Please repull the latest version of this branch with `git pull`

2. Drop the database and Recreate it.

3. Delete `config.php` and `database.php` in `application/config/`

4. Delete Browser Cookies

5. Restart the installer. On the progress site you should now see a button to show the install-logs for debugging

**Reply by HB9HIL** [reply]
Oh and delete the file  `install/.lock`

**Reply by HB9HIL** [reply]
Before starting the installer again enable Logging in **Tab 3** -> Advanced Settings

Error Logging -> Level 1

Should be enough


Then we can see of in `application/logs/` are any error messages

**Reply by BG6HJE-edu** [reply]
Ok, let me try.

**Reply by BG6HJE-edu** [reply]
I delete all the files, recreate the database, restart nginx, delete all cookies and run `git clone https://github.com/HB9HIL/wavelog.git` . This time I get 404 when I open the `http://<ip>/install` in my broswer. But the ICON is displayed. That's weird. 
![404 Error Screenshot](https://github.com/user-attachments/assets/71386984-0859-4555-8a04-24b32ff7d967)

**Reply by HB9HIL** [reply]
I don't use nginx very much, so I can't give an advise here. Maybe you try it with apache2 and it's default config?

**Reply by HB9HIL** [reply]
but this seems to me like a wrong nginx config, and nothing Installer related...

**Reply by BG6HJE-edu** [reply]
Tnx, I'll try again sometime later.

**Reply by HB9HIL** [reply]
Seems like you fixed the installer issue

https://x.com/BG6HJE/status/1816501446214652244

Can you tell the community what the issue was?

**Reply by BG6HJE-edu** [reply]
Okay. I suspect the 404 error is just an Nginx configuration issue. The database error might be a user permission problem. I noticed that even when setting `'root'@'%'` , logging into MySQL still gives the error message `error, … 'root'@'localhost'…` . So, this time I created a new MySQL user called "wavelog" and granted it permissions, instead of using "root" directly.

**Reply by BG6HJE-edu** [reply]
Currently, there are still some issues with log analysis, but the problems discussed here have been resolved. Perhaps I should create a new discussion specifically for log issues later, or wait for a new version release to see if the problem persists.

**Reply by HB9HIL** [reply]
> Currently, there are still some issues with log analysis, but the problems discussed here have been resolved. Perhaps I should create a new discussion specifically for log issues later, or wait for a new version release to see if the problem persists.

What kind of issue with log analysis?
